spec/services/course/user_registration_service_spec.rb

```diff
@@ -16,6 +16,15 @@ def self.registration_with_invitation_code
       end
     end
 
+    def self.registration_with_invitation_code_and_role
```

Comment:
what's the purpose of defining this method?  you can shift the corresponding `let` to the context that you need.

Reply:
Ok, will move it.